app/javascript/gobierto_data/webapp/components/sets/data/SQLEditorHeader.vue

```diff
@@ -56,6 +56,7 @@
       :placeholder="labelQueryName"
       :value="queryName"
       @save="onSaveEventHandler"
+      @disable-key-events="removeKeyboardListener"
```

Comment:
Yes, you're trying an `stopPropagation` (2/2)

Reply:
Fixed, I tried to use `stopPropagation` but doesn't work with `keyup`, with `keyup` the event is already fired.

https://github.com/PopulateTools/gobierto/blob/3813890173cd667de1714e76c04de0efe69c0dbe/app/javascript/gobierto_data/webapp/components/commons/SavingDialog.vue#L12

I've changed it for `keydown`

Reply:
I had to set as well the events for the EditorCode/Header as `keydown` because this:
https://github.com/PopulateTools/gobierto/blob/3813890173cd667de1714e76c04de0efe69c0dbe/app/javascript/gobierto_data/webapp/components/sets/data/SQLEditorCode.vue#L107-L108

To make the `stopPropagation` work, both events should be the same, otherwise nothing will happen. That is, both elements must listen `keydown` or `keyup` or whatever. Also, the keyCodes are different depending on the event!! 🤦‍♀️ 🤦‍♂️ 🤷‍♀️ 🤷‍♂️
Keep care of that!

Reply:
Have you deprecated the `disable-key-events`? If so, remove dead-code